src/main/resources/executables/db-dump-2.txt

```diff
@@ -0,0 +1,20 @@
+user_1, $2b$10$AL/XPsWYU0cekug9KxIEBuSWbgaNCjaDWq/w/MBT0l3FEyz1RtaE2
```

Comment:
can you please group these 2 in a folder in executables? db-dumps/db-dump-*.txt

Reply:
Done!